ci/travis/install-bazel.sh

```diff
@@ -121,3 +121,24 @@ build --remote_upload_local_results=false
 EOF
   fi
 fi
+
+if [ "${TRAVIS-}" = true ]; then
+  # Inject the wrapper function to bashrc, so in future invocation
+  # we get a shorthand to export logs to a temp file on each
+  # test/build event.
+  read -r -d '' bazel_wrapper_func <<-'END'
+bazel() {
+  case "${1-}" in
+    --*) echo "Bazel startup options not yet implemented for this thunk" 1>&2; false;;
+    test) command bazel "$1" --build_event_json_file "$(mktemp /tmp/bazel_event_logs/bazel_log.XXXXX)" "${@:2}";;
+    *) command bazel "$@";;
+  esac
+}
+END
+
+  echo "DEBUG: appending func to bashrc"
+  printf "%s\n" "${bazel_wrapper_func}" >> ~/.bashrc
```

Comment:
```suggestion
  bazel() {
    case "${1-}" in
      --*) echo "Bazel startup options not yet implemented for this thunk" 1>&2; false;;
      test) command bazel "$1" --build_event_json_file "$(mktemp /tmp/bazel_event_logs/bazel_log.XXXXX)" "${@:2}";;
      *) command bazel "$@";;
    esac
  }
  declare -f bazel >> ~/.bashrc
```
You can take out the `read` and just use `declare -f`.

Reply:
@simon-mo Actually, you could generalize this by just creating your own `ci/travis/.bashrc` and appending that to `~/.bashrc`. That way you don't have to manually dump everything individually.

Reply:
I see. That's true, the added section is independent of the install-bazel script. Thanks a lot!!!

Reply:
Yup!